idioms/angularNestJsLambda/backend/models/entity/options/auto-increment.idiom

```diff
@@ -0,0 +1 @@
+@AutoIncrement
```

Comment:
We will need to review all the options idioms.
They are at a very low level.
It is better to use more business-level concepts such as a primary key idiom with both autoincremet and primarykey decorators.